contrib/cloud.google.com/go/pubsub.v1/pubsub_test.go

```diff
@@ -86,6 +86,72 @@ func TestPropagation(t *testing.T) {
 	}, spans[2].Tags())
 }
 
+func TestPropagationWithServiceName(t *testing.T) {
```

Comment:
This is just a copy paste of another test. We need a test that tests only one thing: that the service name is overridden. The rest is for other tests to validate.